src/parameterList.ts

```diff
@@ -1,45 +1,43 @@
-  articleListToIgnore: 'List of articles to ignore. Can be a comma seperated list of titles or a local path or http(s) URL to a file with one title (in UTF8) per line',
-  customZimFavicon: 'Use this option to give a path to a PNG favicon, it will be used in place of the Mediawiki logo. This can be a local path or an HTTP(S) url',
-  customZimTitle: 'Allow to configure a custom ZIM file title.',
-  customZimDescription: 'Allow to configure a custom ZIM file description. Max length is 80 chars.',
-  customZimLongDescription: 'Allow to configure a custom ZIM file long description. Max length is 4000 chars.',
-  customZimTags: 'Allow to configure custom ZIM file tags (semi-colon separated).',
-  customZimLanguage: 'Allow to configure a custom ISO639-3 content language code.',
+  mwUrl: 'MediaWiki base URL (all URL paths appending)',
+  adminEmail: 'Email of the MWoffliner operator. Will be put in the HTTP user-agent string for information only',
+  articleList: 'List of articles to include. Can be a comma separated list of titles or a local path or http(s) URL to a file with one title (in UTF8) per line',
+  articleListToIgnore: 'List of articles to ignore. Comma separated list of titles or local path or HTTP(S) URL to a file with one title (in UTF8) per line',
+  customZimFavicon: 'Path/URL to a PNG favicon (will be resized to 49x49). Default to MediaWiki if not set',
+  customZimTitle: 'ZIM custom title metadata (30 characters max)',
+  customZimDescription: 'ZIM custom description (80 characters max)',
+  customZimLongDescription: 'ZIM custom long description (4000 characters max)',
+  customZimTags: 'ZIM tags metadata (semi-colon separated)',
+  customZimLanguage: 'ZIM ISO639-3 content language code',
   customMainPage: 'Allow to configure a custom page as welcome page.',
-  filenamePrefix: 'For the part of the ZIM filename which is before the format & date parts.',
+  filenamePrefix: 'Part of the ZIM filename which is before the format & date parts.',
   format:
-    'Specify a flavour for the scraping. If missing, scrape all article contents. Each --format argument will cause a new local file to be created but options can be combined. Supported options are:\n * novid: no video & audio content\n * nopic: no pictures (implies "novid")\n * nopdf: no PDF files\n * nodet: only the first/head paragraph (implies "novid")\nFormat names can also be aliased using a ":"\nExample: "... --format=nopic:mini --format=novid,nopdf"',
+    'Specify a flavour for the scraping. If missing, scrape all article contents. Each --format argument will cause a new local file to be created but options can be combined. Supported options are:\n * novid: no video & audio content\n * nopic: no pictures (implies "novid")\n * nopdf: no PDF files\n * nodet: only the first/head paragraph (implies "novid")\nFlavour can be named (and corresponding ZIM metadata will be created) using a ":":\nExample: "--format=nopic,nodet:mini"',
```

Comment:
Flavour can be named (and corresponding ZIM metadata will be created) using a ":":\nExample: "--format=nopic,nodet:mini"'

=> 

Flavour can be aliased using a ":" followed by the alias, and this alias will be used instead of the flavour for ZIM metadata (ZIM filename, name, ...):\nExample: "--format=nopic,nodet:mini"'

(I find the original alias word more meaningful / easy to use and read, but I might be wrong)

Reply:
Not sure your proposal is appropriate. If no flavour/alias specified, than the "Flavour" Metadata will be empty. An alias assumes there is already a primary name which here won't be the case.

Reply:
My proposal is only for the last phrase, everything before must be kept. Regarding the renaming, it would be good, but this is a breaking change ...

Reply:
Keeping like this for the moment, not sure myself how to deal with this "unclear" situation